### ISSUE TITLE
Parse Dashboard does not load.

### ISSUE THREAD

**Issue by KJDEV-PL** [bitnami/bitnami-docker-parse-dashboard#39]
When you enter http://MY-IP/ dashboard, it won't load.
![opera-screen1](https://user-images.githubusercontent.com/48406825/68902185-e84a4a80-0737-11ea-8246-4956aae9264e.png)
I checked on several browsers and each is the same.

I use Portainer.io
Docker-compose file:
[parse.txt](https://github.com/bitnami/bitnami-docker-parse-dashboard/files/3849003/parse.txt)

> version: '2'
> services:
>   mongodb:
>     image: 'bitnami/mongodb:4.0'
>     volumes:
>       - 'mongodb_data:/bitnami'
>   parse:
>     image: 'bitnami/parse:3'
>     ports:
>       - '1337:1337'
>     environment:
>       - PARSE_HOST=127.0.0.1
>     volumes:
>       - 'parse_data:/bitnami'
>     depends_on:
>       - mongodb
>   parse-dashboard:
>     image: 'bitnami/parse-dashboard:2'
>     ports:
>       - '80:4040'
>     environment:
>       - PARSE_HOST=127.0.0.1
>     volumes:
>       - 'parse_dashboard_data:/bitnami'
>     depends_on:
>       - parse
> volumes:
>   mongodb_data:
>     driver: local
>   parse_data:
>     driver: local
>   parse_dashboard_data:
>     driver: local

**Comment by thanakijwanavit** [comment]
same problem here

**Comment by andresbono** [comment]
Hi @KJDEV-PL, @thanakijwanavit, I couldn't reproduce the issue. I launched the docker-compose that you provided and then I accessed to http://127.0.0.1:

![image](https://user-images.githubusercontent.com/1057973/68945745-672f9980-07b1-11ea-84f2-0bf795c4d489.png)

I could login with default credentials:

![image](https://user-images.githubusercontent.com/1057973/68945799-829aa480-07b1-11ea-93f9-5217dafe21bb.png)

Are you using the loopback address too? Does the console of your web browser show any error messages?

**Comment by KJDEV-PL** [comment]
I have it all installed on VPS from OVH, with one IPv4 address and all ports unblocked, without any firewall.  After entering the external IP address, he enters the login panel, where you can log in, and then load nothing :(

**Comment by andresbono** [comment]
Note that the web browser console shows that the destination of the POST query is `127.0.0.1` but it should actually be the IP address of your OVH VPS.

In order to change that, modify the `PARSE_HOST` environment variable of your docker-compose. If you do a start from scratch (that means `docker-compose down -v`, that will remove the persisted volumes), you should get the problem solved.

If you don't want to remove the volumes, you will need to modify the `/bitnami/parse-dashboard/config.json` file in the container and restart the containers.

@thanakijwanavit, I hope this information is also useful for you.

**Comment by evleite** [comment]
I got this problem, today, I just run docker-compose up -d. 
Everything on the logs is good, but I cannot access the dashboard.
On Chorme console I got a feel problems:

dashboard.bundle.js:82 POST http://172.19.0.3:1337/parse/serverInfo net::ERR_CONNECTION_TIMED_OUT
dashboard.bundle.js:82 POST http://172.19.0.3:1337/parse/schemas net::ERR_CONNECTION_TIMED_OUT

I am accessing the dashboard via localhost

Thanks in advance.
